torch/_inductor/codegen/wrapper.py

```diff
@@ -2401,6 +2408,75 @@ def codegen_unbacked_symbol_decl(self, symbol):
             self.unbacked_symbol_decls.add(name)
             return self.declare + name
 
+    def codegen_unbacked_symbol_defs_for_outputs(
```

Comment:
Code movement + small refactoring (parametrize it with output_name, outputs and unbacked_bindings)